Entities/Vehicles/Common/VehicleConvert.as

```diff
@@ -1,216 +1,233 @@
+			if (attackersCount > friendlyCount)
+			{
+				ticks--;
+			}
+			//un-convert gradually
+			else if (attackersCount < friendlyCount || attackersCount == 0)
+			{
+				ticks = Maths::Min(ticks + 1, GetCaptureTime(this));
+			}
+
+			this.set_s16(counter_prop, ticks);
+			this.Tag(raid_tag);
+
+			if (ticks <= 0)
+			{
+				this.server_setTeamNum(attackerTeam);
+				reset_timer = true;
+			}
+			else
+			{
+				this.set_s16(friendly_prop, friendlyCount);
+				this.set_s16(enemy_prop, attackersCount);
+				reset_timer = false;
+			}
+
+			sync = true;
+		}
+	}
+	else
+	{
+		this.Untag(raid_tag);
+	}
+
+	if (reset_timer)
+	{
+		this.set_s16(friendly_prop, 0);
+		this.set_s16(enemy_prop, 0);
+
+		this.set_s16(counter_prop, GetCaptureTime(this));
+		this.Untag(raid_tag);
+		sync = true;
+	}
+
+	if (sync)
+	{
+		this.Sync(friendly_prop, true);
+		this.Sync(enemy_prop, true);
+
+		this.Sync(counter_prop, true);
+		this.Sync(raid_tag, true);
+	}
+
+}
+
+void onChangeTeam(CBlob@ this, const int oldTeam)
+{
+	if (this.getTeamNum() >= 0 && this.getTeamNum() < 10)
+	{
+		CSprite@ sprite = this.getSprite();
+		if (sprite !is null)
+		{
+			sprite.PlaySound("/VehicleCapture");
+		}
+
+		ConvertPoint(this, "VEHICLE");
+		ConvertPoint(this, "DOOR");
+	}
+}
+
+void ConvertPoint(CBlob@ this, const string pointName)
+{
+	AttachmentPoint@ point = this.getAttachments().getAttachmentPointByName(pointName);
+	if (point !is null)
+	{
+		CBlob@ blob = point.getOccupied();
+		if (blob !is null)
+		{
+			blob.server_setTeamNum(this.getTeamNum());
+		}
+	}
+}
+
+int GetCaptureTime(CBlob@ blob)
+{
+	if (blob.hasTag(short_raid_tag))
+	{
+		return short_capture_half_seconds;
+	}
+	return capture_half_seconds;
+}
+
+
+// alert and capture progress bar
+
+void onRender(CSprite@ this)
+{
+	if (g_videorecording)
+		return;
+
+	CBlob@ blob = this.getBlob();
+	CCamera@ camera = getCamera();
+	if (blob is null || !blob.hasTag(raid_tag))
+		return;
+
+	Vec2f pos2d = getDriver().getScreenPosFromWorldPos(blob.getPosition());
+
+	s16 friendlyCount = blob.get_s16(friendly_prop);
+	s16 enemyCount = blob.get_s16(enemy_prop);
+
+	f32 hwidth = 45 + Maths::Max(0, Maths::Max(friendlyCount, enemyCount) - 3) * 8;
+	f32 hheight = 30;
+
+	if (camera.targetDistance > 0.9)
+	{
+	 //draw capture bar
+	 pos2d.y -= 40;
+	 f32 padding = 4.0f;
+	 f32 shift = 29.0f;
+	 s32 captureTime = blob.get_s16(counter_prop);
+	 f32 progress = (1.1f - float(captureTime) / float(GetCaptureTime(blob)))*(hwidth*2-13);
+	 GUI::DrawPane(Vec2f(pos2d.x - hwidth + padding, pos2d.y + hheight - shift - padding),
+	 			   Vec2f(pos2d.x + hwidth - padding, pos2d.y + hheight - padding),
+				   SColor(175,200,207,197));
+	if (progress >= float(8)) //dont draw progress if capture cant start
+	{
+	 GUI::DrawPane(Vec2f(pos2d.x - hwidth + padding, pos2d.y + hheight - shift - padding),
+				   Vec2f((pos2d.x - hwidth + padding) + progress, pos2d.y + hheight - padding),
+				   SColor(175,200,207,197));
+	}
+	print(""+progress);

```

Comment:
Remove the debug print and fix the indentation around this area, you have a tab and a 1 space bar for everything inside the if statement.